core/src/apps/cardano/helpers/__init__.py

```diff
@@ -1,6 +1,8 @@
-LOVELACE_MAX_SUPPLY = 45_000_000_000 * 1_000_000
-INPUT_PREV_HASH_SIZE = 32
-ADDRESS_KEY_HASH_SIZE = 28
-SCRIPT_HASH_SIZE = 28
-OUTPUT_DATUM_HASH_SIZE = 32
-SCRIPT_DATA_HASH_SIZE = 32
+from micropython import const
+
+LOVELACE_MAX_SUPPLY = const(45_000_000_000 * 1_000_000)
+INPUT_PREV_HASH_SIZE = const(32)
+ADDRESS_KEY_HASH_SIZE = const(28)
+SCRIPT_HASH_SIZE = const(28)
+OUTPUT_DATUM_HASH_SIZE = const(32)
+SCRIPT_DATA_HASH_SIZE = const(32)
```

Comment:
does using `const` here help at all, if the constants need to be imported at usage sites anyway?

Reply:
Deleting the `const` made no binary difference whatsoever.

> if the constants need to be imported at usage sites anyway

Does this play any role here? The majority of constants are defined in one file and imported/used in other file

Reply:
The purpose of `const` is to enable build-time optimization, as in, the bytecode compiler will insert the literal value whenever you use the constant in code.
But this seems to only work within a single file, so constants that are imported elsewhere are not affected. And because there are no usages in this file, changing the values to const is pure churn.

Reply:
Right, deleted `const`s in [389478d](https://github.com/trezor/trezor-firmware/pull/2525/commits/389478d0562c79c944fee7a789004f7505ca0430)